### PR TITLE
[DE] Update `osu! wiki/Maintenance`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Translation notes

- not sure if my translation of line 102 is correct. The line in combination with the three bullet points below seems odd to me. Maybe I'm just too tired. I or someone else should take a second look with a fresh mind.
- `Markdown "rule" remark lint rule` was not easy to translate. I've settled for `Lint-Regel "Regel" von remark für Markdown`. If that sounds too chunky, I could change it to `Markdown-Lint-Regel "Regel" von remark`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_wiki/Maintenance/de.md [hunk 1]
```diff
@@ -50,14 +44,72 @@ Bedenke, dass beim Hinzufügen von Weiterleitungen für neue oder bereits existi
...
+| 4 | Kaputte Wiki-Links | Befehl `check-links` aus [`osu-wiki-tools`](https://github.com/Walavouchey/osu-wiki-tools) | Ob interne [Wiki-Links](/wiki/Article_styling_criteria/Formatting#wiki-links) auf einen tatsächlichen Artikel, Newsbeitrag (für Newsbeitrag-Links) oder einen Abschnitt davon zeigen. | Füge `SKIP_WIKILINK_CHECK` irgendwo zur Beschreibung des Änderungsvorschlags hinzu. |
+| 5 | Nicht mehr aktuelle Übersetzungen | Befehl `check-outdated-articles` aus [`osu-wiki-tools`](https://github.com/Walavouchey/osu-wiki-tools) | Ob Übersetzungen korrekt als [nicht mehr aktuell gekennzeichnet](/wiki/Article_styling_criteria/Formatting#nicht-mehr-aktuelle-übersetzungen) werden, wenn ein englischer Artikel aktualisiert wird. | Füge `SKIP_OUTDATED_CHECK` irgendwo zur Beschreibung des Änderungsvorschlags hinzu. |
+
+##### Lint-Regel [`no-heading-punctuation`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-heading-punctuation) von remark für Markdown
```
Comment (respektive) [inline wiki/osu!_wiki/Maintenance/de.md:72]: seems fine to me

File: wiki/osu!_wiki/Maintenance/de.md [hunk 2]
```diff
@@ -50,14 +44,72 @@ Bedenke, dass beim Hinzufügen von Weiterleitungen für neue oder bereits existi
...
+
+Titel sind im Allgemeinen keine vollständigen Sätze. Dadurch treten Satzzeichen am Ende von Überschriften normalerweise nicht auf und sind auch nicht erlaubt. Satzzeichen am Ende des Titels werden jedoch manchmal gebraucht, da sie beispielsweise in Songtiteln oder Künstlernamen enthalten sind.
```
Comment (respektive) [inline wiki/osu!_wiki/Maintenance/de.md:74]:
```suggestion
Titel sind im Allgemeinen keine vollständigen Sätze. Dadurch treten Satzzeichen am Ende von Überschriften normalerweise nicht auf und sind auch nicht erlaubt. Satzzeichen am Ende des Titels werden jedoch manchmal gebraucht, wenn sie beispielsweise in Songtiteln oder Künstlernamen enthalten sind.
```

File: wiki/osu!_wiki/Maintenance/de.md [hunk 3]
```diff
@@ -69,6 +121,8 @@ Einige Features sind nicht direkt mit der osu!-Webseite verwandt, können aber n
 
 - [osu-wiki status](https://osu.wiki/status/de): Listet alle Artikel pro Sprache auf und die Kategorie der Instandhaltung, die sie benötigen (Übersetzung, Aktualisierung, Erweiterung von unvollständigen Artikeln). Siehe [ppy/osu-wiki#2486](https://github.com/ppy/osu-wiki/issues/2486) für Funktionalitätswünsche.
 - [osu-wiki-bin](https://github.com/cl8n/osu-wiki-bin): ein Node.js-Hilfsprogramm für automatische Überprüfungen und Bearbeitungen (defekte Links, Aktualisierungen in Nutzergruppen, Textersetzungen im gesamten Wiki und andere Dinge)
+- [osu-wiki-tools](https://github.com/Walavouchey/osu-wiki-tools): ein Python-Tool für automatisierte Überprüfungen (kaputte Links, nicht mehr aktuelle Artikel), die in der CI verwendet werden
+- [scissors](https://github.com/TicClick/scissors): ein Rust-Tool zur Überprüfung von Unstimmigkeiten zwischen Benutzernamen und Länderkennzeichen in Benutzerlinks
```
Comment (respektive) [inline wiki/osu!_wiki/Maintenance/de.md:125]:
```suggestion
- [scissors](https://github.com/TicClick/scissors): ein Rust-Tool zur Überprüfung von Unstimmigkeiten zwischen Benutzernamen und Länderflaggen in Benutzerlinks
```

**Review by respektive — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
Thank you 👍

**Review by respektive — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**